epub32/spec/epub-packages.html

```diff
@@ -3460,42 +3460,30 @@ <h1>EPUB Navigation Document</h1>
-						href="https://www.w3.org/TR/html/sections.html#the-nav-element"><code>nav</code></a> element.
-					Each <code>nav</code> element in an EPUB Navigation Document represents a data island — an embedded
-					source of specialized information within the general markup — from which Reading Systems can
-					retrieve navigational information. Unlike typical XML data islands, however, the information within
-					the <code>nav</code> element remains human readable as an [[HTML]] document.</p>
-
-				<p>To facilitate machine readability, the content model of <code>nav</code> elements in EPUB Navigation
-					Documents is restricted relative to what is allowed in general XHTML Content Documents.</p>
-
-				<p>Note that the EPUB Navigation Document is not exclusively for machine processing. Formulating the
-					document as an XHTML Content Document enables its reuse in the linear reading order, avoiding the
-					creation of additional tables of contents (i.e., it can also be added to the <a
-						href="#elemdef-opf-spine">spine</a>). The visual display of components defined in the EPUB
-					Navigation Document can be controlled using the <a href="#sec-package-nav-def-hidden">hidden</a>
-					attribute, which has no effect outside of spine rendering (i.e., hiding a component from rendering
-					in the spine does not hide it from Reading System presentation in a custom control).</p>
-
-				<p>When designing an EPUB Navigation Document for such dual use, however, be aware that machine
-					extraction of the content can result in loss of formatting control. Scripting, styling and other
-					HTML formatting can be stripped by a Reading System as it generates a custom control, such as the
-					table of contents, from the markup. If such formatting and functionality is used, then the EPUB
-					Navigation Document also needs to be included in the linear reading order. Another design
-					consideration is to use <a href="epub-contentdocs.html#confreq-cd-scripted-spine">progressive
-						enhancement</a> [[ContentDocs32]] techniques for scripting and styling of the navigation
-					document, such that that the content will retain its integrity when rendered in a non-browser
-					context.</p>
+				<p>The EPUB Navigation Document is an <a>XHTML Content Document</a>,
+				 	but with additional restrictions on its structure to facilitate the machine-processing of its contents.
+					[[HTML]] <a
+					href="https://www.w3.org/TR/html/sections.html#the-nav-element"><code>nav</code></a> elements 
+					contain the specialized navigational information, which remains human-readable 
+					as well as allowing Reading Systems to generate navigational interfaces.</p>
+
+				<p>But the EPUB Navigation Document is not exclusively for machine processing. Because it is 
+					an XHTML Content Document, it can be part of the linear reading order, avoiding the
+					avoiding the need for duplicate tables of contents.
```

Comment:
I salute the use of duplication while talking about the need to avoid duplication, but one "avoiding the" needs to go... ;)